src/impl/tpls/KokkosBlas_Host_tpl.cpp

```diff
@@ -249,6 +249,23 @@ extern "C" {
                                      const std::complex<double>*,
                                      const std::complex<double>*, int*,
                                      /* */ std::complex<double>*, int* );
+
+  ///
+  /// Gesv
+  ///
+
+  void F77_BLAS_MANGLE(sgesv,SGESV)( int*, int*,
+                                     float*, int*, int*,
+                                     float*, int*, int* );
+  void F77_BLAS_MANGLE(dgesv,DGESV)( int*, int*,
+                                     double*, int*, int*,
+                                     double*, int*, int* );
+  void F77_BLAS_MANGLE(cgesv,CGESV)( int*, int*,
+                                     std::complex<float>*, int*, int*,
+                                     std::complex<float>*, int*, int* );
+  void F77_BLAS_MANGLE(zgesv,ZGESV)( int*, int*,
+                                     std::complex<double>*, int*, int*,
```

Comment:
Sometimes compilers warn that `extern "C"` declarations can't accept `std::complex<T>`.  If that should happen, you could always have these functions take `void*` instead of `std::complex<T>*`.  That is what the BLAS' C binding does.

Reply:
Thanks @mhoemmen. I am just afraid if I make change, we also need changes in other BLAS functions. After talking with @kyungjoo-kim , I think I prefer this way, just to make it consistent with other BLAS functions. Moreover, so far we have not seen any binding issue during our spotchecks. If users have any issue with these, I am willing to help them.